copulas/multivariate/vine.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, vine_type, *args, **kwargs):
         """
         super().__init__(*args, **kwargs)
         self.vine_type = vine_type
-        self.u_matrix = None
+        self.u_matrix = None # TODO: Decide whether this should be an attribute or not
```

Comment:
Why is the univariate matrix stored as an attribute? First, I think that a "copula" shouldn't even be responsible transforming the data to produce univariate marginals. However, even if it responsible for it, why is it storing it as an instance attribute?